spec/features/notes_on_merge_requests_spec.rb

```diff
@@ -72,9 +77,9 @@
 
         it "should show the note edit form and hide the note body" do
           within("#note_#{note.id}") do
-            find(".current-note-edit-form", visible: true).should be_visible
-            find(".note-edit-form", visible: true).should be_visible
-            find(:css, ".note-text", visible: false).should_not be_visible
+            expect(find(".current-note-edit-form", visible: true)).to be_visible
+            expect(find(".note-edit-form", visible: true)).to be_visible
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.